static/sass/main.scss

```diff
@@ -202,21 +196,27 @@ header {
 }
 
 footer {
-  @include gradient-bar-bg;
   font-size: 12px;
+  background: $card-background;
   box-shadow: 0 -2px 5px $bar-shadow-color;
   display: flex;
   flex-direction: column;
   justify-content: center;
   text-align: center;
   position: fixed;
+  padding: 8px;
   bottom: 0;
   left: 0;
   right: 0;
   z-index: 3;
 
   div {
     margin-top: $content-padding / 4;
+
+    a + a, a + b, b + a {
```

Comment:
`& > * + *`?

Reply:
Much better.  I should have that one memorized.